src/src/com/microsoft/aad/adal/DateTimeAdapter.java

```diff
@@ -50,12 +50,31 @@
             DateFormat.DEFAULT);
 
     private final DateFormat iso8601Format = buildIso8601Format();
+    private final DateFormat enUs24HourFormat = buildEnUs24HourDateFormat();
+    private final DateFormat local24HourFormat = buildLocal24HourDateFormat();
 
     private static DateFormat buildIso8601Format() {
         DateFormat iso8601Format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'", Locale.US);
```

Comment:
I actually don't think it matters here. What we eventually return back to caller is Date, specific moment in time, with milliseconds in precision, and always UTC. We we need here is just a consistent way to persist this data when serializing/deserializing it. Before 1.1.14, i think we simply use the default format, and default locale. The two different locals is to support backward compatibility.